src/servers/src/http/handler.rs

```diff
@@ -208,7 +208,7 @@ pub async fn sql_format(
 
     let mut parts: Vec<String> = Vec::with_capacity(stmts.len());
     for stmt in stmts {
-        let mut s = format!("{:#}", stmt);
```

Comment:
will this cause different display result due to [alternate](https://doc.rust-lang.org/std/fmt/struct.Formatter.html#method.alternate) become false?

Reply:
Pretty format was not expected. Actually the "pretty" was not working until this sqlparser version (v0.58.0): https://github.com/apache/datafusion-sqlparser-rs/pull/1847

Pretty format now will display too many new lines, breaking the test "test_sql_format_api". So I think it's better to keep the behavior.